core/dbt/cli/main.py

```diff
@@ -64,13 +62,9 @@ class dbtRunnerResult:
 class dbtRunner:
     def __init__(
         self,
-        project: Project = None,
-        profile: Profile = None,
```

Comment:
Does removing these lines break anything outside of `dbt-core`?

Reply:
No, did a full look up on places I know uses this runner and they don't use it at all.